flexible/java-11/springboot-helloworld/src/main/appengine/app.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: java
+env: flex
+runtime_config:
+  operating_system: ubuntu18
+  runtime_version: 11
+handlers:
+- url: /.*
+  script: this field is required, but ignored
```

Comment:
Not sure if this is still true. Please verify when you have time.

Reply:
gcloud app deploy fails if "script" is not present.
ERROR: (gcloud.app.deploy) An error occurred while parsing file: [/tmp/tmpmobgi_d0/app.yaml]
Unknown url handler type.
<URLMap 
    static_files=None 
    upload=None 
    application_readable=None 
    static_dir=None 
    mime_type=None 
    expiration=None 
    require_matching_file=None 
    http_headers=None 
    position=None 
    api_endpoint=None 
    redirect_http_response_code=None 
    url=/.* 
    login=optional 
    auth_fail_action=redirect 
    secure=default 
    script=None
    >
  in "/tmp/tmpmobgi_d0/app.yaml", line 24, column 1